### PR TITLE
Error pages

### PR DESCRIPTION
**Pull request by IGORnvk**
403, 404 and 500 error pages.

### CODE REVIEW

**Comment by v-stamenova** [comment]
You can merge with development to check how it would look like with the new font, but I'd rather have @TimKardol review it

**Comment by v-stamenova** [comment]
@IGORnvk we may need 401 as well, sorry forgot about it

**Comment by IGORnvk** [comment]
Done @v-stamenova

File: resources/views/errors/401.blade.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@extends('errors::minimal')
+
+@section('code', '401')
+@section('message')
+    Access Denied! This zone is exclusive to authorized attendees.<br>
+    Return to the <a href="/" class="text-yellow-300 hover:text-yellow-500 hover:border-b-2 hover:border-yellow-500 transition-all">conference home page</a> to find your path to knowledge and innovation,<br>
+    or ensure you've obtained the right credentials for entry.
```
Comment (TimKardol) [inline resources/views/errors/401.blade.php:7]: You could link them to either registration and/or login. This way it's more helpful.

File: resources/views/errors/401.blade.php [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@extends('errors::minimal')
+
+@section('code', '401')
+@section('message')
+    Access Denied! This zone is exclusive to authorized attendees.<br>
```
Comment (TimKardol) [inline resources/views/errors/401.blade.php:5]: Minor nitpick but change it to:
```suggestion
    Access Denied! This page is exclusive to authorized attendees.<br>
```

File: resources/views/errors/403.blade.php [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@extends('errors::minimal')
+
+@section('code', '403')
+@section('message')
+    Whoa there, Tech Explorer! This path is reserved for authorized eyes only.<br>
+    Our IT Conference's content vault is securely locked.<br>
+    Return to the <a href="/" class="text-yellow-300 hover:text-yellow-500 hover:border-b-2 hover:border-yellow-500 transition-all">main page</a> to access the open zones of knowledge and innovation.
```
Comment (TimKardol) [inline resources/views/errors/403.blade.php:7]: Give the option to return to the homepage or their own dashboard if they are logged in.
I don't know the specific urls for them
```suggestion
    Return to the <a href="/dashboard" class="text-yellow-300 hover:text-yellow-500 hover:border-b-2 hover:border-yellow-500 transition-all">homepage</a> to access the whole website.
```

File: resources/views/errors/404.blade.php [hunk 4]
```diff
@@ -0,0 +1,8 @@
+@extends('errors::minimal')
+
+@section('code', '404')
+@section('message')
+    Lost in the Digital Maze? You've reached a page that's on a coffee break.<br>
```
Comment (TimKardol) [inline resources/views/errors/404.blade.php:5]: Have to tell the client why they are on this page.
```suggestion
    Lost in the Digital Maze? You've reached a page that does not exist.<br>
```

File: resources/views/errors/404.blade.php [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+    While this content isn't here, our IT Conference is bustling with tech excitement.<br>
+    Head back to the <a href="/" class="text-yellow-300 hover:text-yellow-500 hover:border-b-2 hover:border-yellow-500 transition-all">homepage</a> to explore the latest trends and sessions.
```
Comment (TimKardol) [inline resources/views/errors/404.blade.php:7]:
```suggestion
    Head back to the <a href="/" class="text-yellow-300 hover:text-yellow-500 hover:border-b-2 hover:border-yellow-500 transition-all">homepage</a> to explore the latest companies and speakers.
```

**Review by TimKardol — CHANGES_REQUESTED** [review]

**Review by TimKardol — APPROVED** [review]

**Pull request merged**